io.github.sil-car.SqueezeVid.yaml

```diff
@@ -0,0 +1,59 @@
+# https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest
+app-id: io.github.sil-car.SqueezeVid
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: squeeze-vid
+modules:
+  - name: libsvtav1
+    # Ref: https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Build-Guide.md#1-build-and-install-svt-av1
+    sources:
+      - type: git
+        url: https://gitlab.com/AOMediaCodec/SVT-AV1.git
+        tag: v1.4.1
+    # source-depth: 1 "shallow clone" is used by default
+    buildsystem: cmake
+    cmake-args: # undocumented?
+      - -G"Unix Makefiles"
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/usr
+
+  - name: ffmpeg
+    sources:
+      - type: git
+        url: git://source.ffmpeg.org/ffmpeg.git
+        branch: 'n6.0'
+    buildsystem: autotools
+    config-opts:
+      - --prefix=/usr
+      - --disable-doc # removes all docs --disable-{htmlpages,manpages,txtpages}
+      - --disable-ffplay
+      - --enable-version3
+      - --enable-gpl
+      - --enable-libass         # subtitles support
+      - --enable-libaom         # AV1 video encoder
+      # - --enable-libsvtav1      # SVT-AV1 video encoder; requires 22.04+ packages libsvtav1{,-dev}; or build from src
+      - --enable-libvpx         # VP9 video encoder
+      - --enable-libx264        # H.264 video encoder
+
+  - name: pip-deps
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network # needed for pip installs from internet
```

Comment:
this is not allowed. You need to use the flatpak-builder-tools to handle python dependencies.

https://github.com/flatpak/flatpak-builder-tools/tree/master/pip